bin/proving-service/src/commands/update_workers.rs

```diff
@@ -68,13 +68,13 @@ impl UpdateWorkers {
         let proxy_config = ProxyConfig::load_config_from_file()?;
 
         // Create the full URL
-        let url = format!("http://{}:{}?{}", proxy_config.host, proxy_config.port, query_params);
+        let url = format!(
+            "http://{}:{}?{}",
+            proxy_config.host, proxy_config.workers_update_port, query_params
+        );
```

Comment:
I think we need to update doc comments a bit - specifically lines 45 - 46 to mention that we are using a different port for this.